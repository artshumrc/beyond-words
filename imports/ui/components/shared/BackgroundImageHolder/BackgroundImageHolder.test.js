import React from 'react';
import renderer from 'react-test-renderer';

// component:
import BackgroundImageHolder from './BackgroundImageHolder';

describe('BackgroundImageHolder', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(<BackgroundImageHolder imgSrc="test" />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
