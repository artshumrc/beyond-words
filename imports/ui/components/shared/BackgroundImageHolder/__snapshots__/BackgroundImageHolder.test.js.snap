// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundImageHolder renders correctly 1`] = `
<div>
  <div
    className="background-image-holder blur-2--no-remove blur-10 remove-blur"
  >
    <img
      alt="A Homer Commentary in Progress - Commentator"
      className="background-image"
      src="test"
    />
  </div>
  <div
    className="block-screen brown"
  />
</div>
`;
